src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,61 @@
+@use "../utlis/variables";
+
+.stopwatch {
+  height: variables.$clock-size;
+  width: variables.$clock-size;
+  background-color: variables.$clock-face-color;
+  border-radius: variables.$clock-radius;
+  position: relative;
+
+  &__center {
+    height: variables.$clock-center-size;
+    width: variables.$clock-center-size;
+    background-color: variables.$clock-center-color;
+    top: calc(variables.$clock-size / 2) - (variables.$clock-center-size / 2);
+    left: calc(variables.$clock-size / 2) - (variables.$clock-center-size / 2);
+    border-radius: variables.$clock-radius;
+    position: absolute;
+  }
+
+  &__minutes {
+    height: variables.$minutes-length;
+    width: variables.$minutes-width;
+    position: absolute;
+    background-color: variables.$minutes-color;
+    left: calc(variables.$clock-size / 2) - (variables.$minutes-width / 2);
+    top: calc(variables.$clock-size / 2) - variables.$minutes-length;
+    border-radius: variables.$clock-arms-radius;
+    transform-origin: 50% 100%;
+    animation: minutes 60s linear infinite;
+  }
+
+  &__seconds {
+    height: variables.$seconds-length;
+    width: variables.$seconds-width;
+    position: absolute;
+    background-color: variables.$seconds-color;
+    left: calc(variables.$clock-size / 2) - (variables.$seconds-width / 2);
+    top: calc(variables.$clock-size / 2) - variables.$seconds-length;
+    border-radius: variables.$clock-arms-radius;
+    transform-origin: 50% 100%;
+    animation: seconds 5s linear infinite;
+  }
+}
+
+@keyframes seconds {
+  from {
+    transform: translate(-50%, 0) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, 0) rotate(360deg);
+  }
+}
+
+@keyframes minutes {
+  from {
+    transform: translate(-50%, 0) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, 0) rotate(360deg);
+  }
+}
```

Comment:
There is no need to use 2 identical animations, here you only need to create one and use it for the two arrows